### ISSUE TITLE
Implement Selection Sort in Python

### ISSUE THREAD

**Issue by sobhanbera** [sobhanbera/algorithms#204]
**Algorithm/Data Structure**: Selection Sort
**Programming Language**: Python
**Difficulty**: ⭐️⭐️

**Description**: 
Implement the Selection Sort algorithm in Python. Selection Sort is a simple comparison-based sorting algorithm that repeatedly selects the minimum element from an unsorted portion of the list and swaps it with the first element in that unsorted portion. Your task is to correctly sort a given list of elements in ascending order using Selection Sort in Python.

**Additional Information**: 
If you are new to Selection Sort or need guidance on implementing it in Python, feel free to ask questions or seek help in the comments. This is a great opportunity for contributors to gain experience in sorting algorithms and algorithmic optimizations in Python.

**Note**: 
Before starting to work on this issue, please comment below to express your interest and let us know if you have any questions or need clarification on the task.

**Comment by parag477** [comment]
I want to work on this issue. Can you assign this to me?

**Comment by sobhanbera** [comment]
@parag477 assigned. let me know if you need any help!

**Comment by DIPANSHI-10** [comment]
Can you please assign this to me

**Comment by parag477** [comment]
I have added the algorithm please check it and merge my PR.

**Comment by parag477** [comment]
Hey please merge my PR

**Comment by MeetD99** [comment]
Hi, I would like to work on this Issue, can you assign it to me please.

**Comment by sobhanbera** [comment]
Hey @DIPANSHI-10 and @MeetD99,
Issues are FCFS basis only. You can choose another issue or can create your own!

**Comment by siya100** [comment]
hey @sobhanbera 
I would like to work on this issue,so please assign it to me
